src/libgenders/genders_parsing.c

```diff
@@ -673,23 +644,33 @@ _genders_open_and_parse(genders_t handle,
    * change our minds later concerning whether and empty genders file
    * is acceptable.
    */
-  int len, errcount = 0, fd = -1, rv = -1, line_count = 1, parsed_nodes = 0;
+  int len, errcount = 0, rv = -1, line_count = 1, parsed_nodes = 0;
   char buf[GENDERS_BUFLEN];
 
   if (!filename || !strlen(filename))
     filename = GENDERS_DEFAULT_FILE;
 
-  if ((fd = open(filename, O_RDONLY)) < 0) 
+  FILE *f = fopen(filename, "r");
```

Comment:
This is only a nit.  When genders was first written, it didn't assume C99 was ok.  Obviuosly, C99 is probably a safe assumption nowadays, so declaring `f` here is fine.  But dunno if we want to declare `f` at the top of the function just for consistency to rest of the code?

Reply:
eh, it's really close to the top of the block anyway.  I don't think I mind.